repos/fdbt-site/src/pages/selectTimeRestrictions.tsx

```diff
@@ -0,0 +1,187 @@
+                                            timeRestrictions.map((item) => (
+                                                <TimeRestrictionCard key={item.id} timeRestriction={item} />
+                                            ))
+                                        ) : (
+                                            <p className="govuk-body govuk-error-message">
+                                                Create a time restriction in operator settings.
+                                            </p>
+                                        )}
+                                    </div>
+                                </div>
+
+                                <div className="govuk-radios__item">
+                                    <input
+                                        className="govuk-radios__input"
+                                        id="no-choice"
+                                        name="timeRestrictionChoice"
+                                        type="radio"
+                                        value="no"
+                                        data-aria-controls="conditional-contact-2"
+                                    />
+                                    <label className="govuk-label govuk-radios__label" htmlFor="no-choice">
+                                        No
+                                    </label>
+                                </div>
+                            </div>
+                        </fieldset>
+                    </div>
+
+                    <input type="submit" value="Continue" id="continue-button" className="govuk-button" />
+                </>
+            </CsrfForm>
+        </TwoThirdsLayout>
+    );
+};
+
+const TimeRestrictionCard = ({ timeRestriction }: { timeRestriction: PremadeTimeRestriction }): ReactElement => {
+    return (
+        <div className="govuk-grid-column-one-half govuk-!-margin-bottom-5">
+            <div className="card">
+                <div className="card__body time-restriction">
+                    <div className="govuk-radios">
+                        <div className="govuk-radios__item card__radio">
+                            <input
+                                className="govuk-radios__input"
+                                id={`${timeRestriction.name}-radio`}
+                                name="timeRestriction"
+                                type="radio"
+                                value={timeRestriction.name}
+                                aria-label={timeRestriction.name}
+                            />
+                            {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+                            <label className="govuk-label govuk-radios__label" />
+                        </div>
+                    </div>
+
+                    <h4 className="time-restriction-title govuk-heading-m govuk-!-padding-bottom-4">
```

Comment:
This code to the bottom is all identical to the cards in GS, is it worth refactoring?